Changelog.md

```diff
@@ -8,7 +8,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Add `VK_KHR_present_wait` extension (#493)
+- Added `VK_KHR_present_wait` device extension (#493)
+
+### Changed
+
+- Device extension `khr::PipelineExecutableProperties` and `khr::TimelineSemaphore` now expose `fn device()` instead of `fn instance()` (#499)
+- Changed `khr::PipelineExecutableProperties::new()` and `khr::TimelineSemaphore::new()` to take `instance` and `device` as arguments (#499)
+
+### Removed
+
+- Removed `device()` function from `khr::DrawIndirectCount`, `khr::DynamicRendering`, `khr::Synchronization2`, `nv::DeviceDiagnosticCheckpoints` and `nv::MeshShader` device extensions (#494)
```

Comment:
`DrawIndirectCount`, `DynamicRendering`, `DeviceDiagnosticCheckpoints`, `MeshShader` did not have a `device()` function before this PR. They were introduced here earlier, and removed later.

Reply:
Fixed